records/2023-06-27.scalability.increasing.send.rates.md

```diff
@@ -0,0 +1,276 @@
+| Code           | [Remove superfluous database calls](#remove-superfluous-database-calls)                 | [#176](https://app.zenhub.com/workspaces/notify-planning-core-6411dfb7c95fb80014e0cab0/issues/gh/cds-snc/notification-planning-core/176)                                                                                                                                                                                                                                                                               |
+| Code           | [Bulk send to AWS SES](#bulk-send-to-aws-ses)                                           |                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| Code           | [Align SMS pipeline with priority queues](#align-sms-pipeline-with-priority-queues)     | [#2](https://app.zenhub.com/workspaces/notify-planning-core-6411dfb7c95fb80014e0cab0/issues/gh/cds-snc/notification-planning-core/2) [#186](https://app.zenhub.com/workspaces/notify-planning-core-6411dfb7c95fb80014e0cab0/issues/gh/cds-snc/notification-planning-core/186)                                                                                                                                          |
+| Testing        | [Improve performance tests and reporting](#improve-performance-tests-and-reporting)     | [#177](https://app.zenhub.com/workspaces/notify-planning-core-6411dfb7c95fb80014e0cab0/issues/gh/cds-snc/notification-planning-core/177)                                                                                                                                                                                                                                                                               |
+| Investigation  | [Fix 50%+ performance decrease in past year](#fix-50-performance-decrease-in-past-year) | [#178](https://app.zenhub.com/workspaces/notify-planning-core-6411dfb7c95fb80014e0cab0/issues/gh/cds-snc/notification-planning-core/178)                                                                                                                                                                                                                                                                               |
```

Comment:
Love this table.. maybe we could add a status column to it. We can take some time together if you want, where one of us verify the issue and the other update the row.

Reply:
yesss that's a good idea. And it looks like most of these issues are still just open so they could use some updating themselves...